app/views/layouts/application.html.erb

```diff
@@ -1,5 +1,6 @@
 <!DOCTYPE html>
-<html lang="<%= FastGettext.locale %>">
+<html lang="<%= FastGettext.locale %>"   style="position: relative;
```

Comment:
try not to put styling directly in the html. place it in a css file instead (use the application.scss for now since we haven't come up with an approach we want to use for bootstrap overrides)
```css
html {
  position: relative;
  min-height: 100%;
}
```